assembl/static2/js/app/graphql/IdeaWithPosts.graphql

```diff
@@ -0,0 +1,41 @@
+query IdeaWithPosts($lang: String!, $id: ID!) {
+  idea: node(id: $id) {
+    ... on Idea {
+      id
+      title(lang: $lang)
+      description(lang: $lang)
+      imgUrl
+      posts {
+        edges {
+          node {
+            ... on Post {
+              id
+              subject
+              body
```

Comment:
This is part of the feature about translating posts 700. We will have a lang variable indeed.